core/primitives/src/transaction.rs

```diff
@@ -1086,59 +552,38 @@ impl fmt::Debug for TransactionResult {
     }
 }
 
-/// Logs for transaction or receipt with given hash.
-#[derive(PartialEq, Clone, Serialize, Deserialize)]
-pub struct TransactionLogs {
-    #[serde(with = "base_format")]
-    pub hash: CryptoHash,
-    pub lines: Vec<LogEntry>,
-    pub receipts: Vec<CryptoHash>,
-    pub result: Option<Vec<u8>>,
-}
-
-impl fmt::Debug for TransactionLogs {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("TransactionLogs")
-            .field("hash", &self.hash)
-            .field("lines", &format_args!("{}", logging::pretty_vec(&self.lines)))
-            .field("receipts", &format_args!("{}", logging::pretty_vec(&self.receipts)))
-            .field("result", &format_args!("{}", logging::pretty_result(&self.result)))
-            .finish()
-    }
-}
-
 /// Result of transaction and all of subsequent the receipts.
 #[derive(PartialEq, Clone, Serialize, Deserialize, Default)]
 pub struct FinalTransactionResult {
     /// Status of the whole transaction and it's receipts.
     pub status: FinalTransactionStatus,
-    /// Logs per transaction / receipt ids ordered in DFS manner.
-    pub logs: Vec<TransactionLogs>,
+    /// Transaction results.
+    pub transactions: Vec<TransactionResult>,
```

Comment:
use TransactionLog that contains TransactionResult + hash of the transaction/receipt.